src/backends/fvm_multicore.hpp

```diff
@@ -117,6 +119,8 @@ struct backend {
 
     using mechanism = mechanisms::mechanism_ptr<backend>;
 
+    using stimulus = mechanisms::multicore::stimulus<backend>;
```

Comment:
The namespaces are a bit of a mess here. To be fixed in "the big namespace rationanlization".